x-pack/plugins/cloud_security_posture/public/components/accounts_evaluated_widget.tsx

```diff
@@ -75,10 +79,10 @@ export const AccountsEvaluatedWidget = ({
           key={benchmark.type}
           onClick={() => {
             if (benchmark.provider) {
-              navToFindingsByCloudProvider(benchmark.provider);
+              return navToFindingsByCloudProvider(benchmark.provider);
             }
             if (benchmark.benchmarkId) {
-              navToFindingsByCisBenchmark(benchmark.benchmarkId);
+              return navToFindingsByCisBenchmark(benchmark.benchmarkId);
```

Comment:
why do we need to return here? no one takes in the value and the functions themselves do not return anything. i think this is redundant.